README.md

```diff
@@ -1,16 +1,17 @@
-# Web Monetization Tools
+# Publisher Tools
 
-<a href="#what-is-web-monetization-tools">
-  <img src="https://github.com/interledger/web-monetization-tools/blob/25fff6ab48b052ac1190cf3734cb96aba99ed9a2/docs/preview.png?raw=true" alt="Web monetization tools preview image">
+<a href="#what-is-the-publisher-tools">
+  <img src="https://github.com/user-attachments/assets/f3776c29-e64d-439d-8321-6b9a03773a9d" alt="publisher tools preview image">
 </a>
 
-## What are Web Monetization Tools?
 
-Web Monetization Tools are marketing tools, that provide website owners with a set of utilities to display payment options as well as information that encourage visitors to use Web Monetization extension alongside payment solution for one-time payments, such as donations.
+## What are the Publisher tools?
```

Comment:
"What are publisher tools" or "What are the publisher tools" (if you go with the first, the href on line 3 needs to be updated.

Reply:
Yes I like that version better